### DISCUSSION TITLE
Nighly builds

### DISCUSSION THREAD

**Discussion by decriptor** [dotnet/dotnet-monitor#2869] [Q&A] [answered]
Is it possible to get nightly builds of dotnet-monitor? There are some new features I'd really like to try out soon than later.

**Comment by jander-msft** [comment, marked as answer]
Yes, you can get them from the [dotnet-tools](https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-tools/NuGet/dotnet-monitor/) public feed. We make no guarantee of the state of these builds as they are typically hot-off-the-press for each change that is made.
